src/screens/FeedScreen/Content.js

```diff
@@ -20,7 +20,7 @@ import {
   POST_TYPE_POLL,
   POST_TYPE_STANDARD
 } from '../../utils/constants';
-import {fonts, normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
+import {fonts, normalizeFontSizeByWidth} from '../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `normalize` function was removed from the import statement. If this function is not used anywhere else in the file, this change is good for code cleanliness and readability.
```diff
- import {fonts, normalize, normalizeFontSizeByWidth} from '../../utils/fonts';
+ import {fonts, normalizeFontSizeByWidth} from '../../utils/fonts';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->